komfydb/execution/seq_scan.cc

```diff
@@ -0,0 +1,52 @@
+#include <string>
+
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+
+#include "komfydb/common/tuple.h"
+#include "komfydb/common/tuple_desc.h"
+#include "komfydb/execution/op_iterator.h"
+#include "komfydb/transaction/transaction_id.h"
+
+namespace komfydb::execution {
+
+SeqScan::SeqScan(TransactionId tid, int table_id, absl::string_view table_alias)
+    : tid(tid), table_id(table_id), table_alias(table_alias) {}
+
+SeqScan::SeqScan(TransactionId tid, int table_id) {
+  SeqScan(tid, table_id, common::GenerateUuidV4());
+}
+
+std::string SeqScan::GetTableName() {
+    ASSIGN_OR_RETURN(std::string table_name, catalog->GetTableName(table_id);
+    return table_name;
+}
+
+std::string SeqScan::GetAlias() {
+  return table_alias;
+}
+
+absl::Status SeqScan::Open() {
```

Comment:
Please see my comment on `TableIterator::Open()`, I feel like it is the same case here.